### DISCUSSION TITLE
What does cpp/dds folder do [13613]

### DISCUSSION THREAD

**Discussion by minrui-hust** [eProsima/Fast-DDS#2442] [Q&A]
it seems sources under this folder is of no use, what are they used for?

**Comment by JLBuenoLopez** [comment]
Hi @minrui-hust,

I have moved your question to our Q&A forum where it is best placed.

The `cpp/dds` folder contains the under development code for the Fast DDS PSM API. You can find some more infor in this [other answer](https://github.com/eProsima/Fast-DDS/discussions/2439#discussioncomment-2050886).

I hope this information is helpful!

**Reply by minrui-hust** [reply]
got it, thanks
